src/components/map/GpxService.js

```diff
@@ -61,6 +59,11 @@ goog.require('ga_styles_service');
             geom = new ol.geom.Point(geom.getCoordinates().slice(0, 2), 'XY');
             feature.setGeometry(geom);
           }
+
+          if (geom && /^XYZ/.test(geom.getLayout())) {
+            geom.set('altitudeMode', 'relativeToGround');
```

Comment:
Not sure, best would be to ask the user what he wants.

It's really for flights. It could look like a bug if we don't clamp to ground stuff that we are supposed to be clamped like hikes.

Reply:
Yes I thought the same, but if the elevation is correct it seems the display looks ok. We'll see if users complain about it.